website/content/en/highlights/2022-11-07-0-26-0-upgrade-guide.md

```diff
@@ -66,3 +67,11 @@ get the same functionality through a `remap` transform with [`geoip` enrichment
 tables](/docs/reference/configuration/global-options/#enrichment_tables.geoip). These can be used
 with [VRL's enrichment table functions](/docs/reference/vrl/functions/#enrichment-functions) to
 enrich events using a [GeoIP database](https://www.maxmind.com/en/geoip2-databases).
+
+#### Enrichment of `fluent` source with `source_type` field {#fluent-source-type-field}
```

Comment:
We typically haven't considered field additions to be breaking changes. I think we can drop this note.

Reply:
Oh good to know... I also made changes to this file for the `vector` source PR. So I can remove that in that PR.